code/modules/client/preference/preferences.dm

```diff
@@ -1888,17 +1922,18 @@ GLOBAL_LIST_INIT(special_role_times, list( //minimum age (in days) for accounts
 								return
 							R.company = choice
 							R = GLOB.all_robolimbs[R.company]
-							if(R.has_subtypes == 1) //If the company the user selected provides more than just one base model, lets handle it.
```

Comment:
This was an intentional choice. Changing `== 1` to `!= 0`.

This is because I reformatted how robolimbs work a bit.